Northstar.Client/mod/scripts/vscripts/ui/menu_ingame.nut

```diff
@@ -85,10 +85,12 @@ void function InitInGameMPMenu()
 	var gameHeader = AddComboButtonHeader( comboStruct, headerIndex, "#MENU_HEADER_GAME" )
 	var leaveButton = AddComboButton( comboStruct, headerIndex, buttonIndex++, "#LEAVE_MATCH" )
 	Hud_AddEventHandler( leaveButton, UIE_CLICK, OnLeaveButton_Activate )
-	#if DEV
-		var devButton = AddComboButton( comboStruct, headerIndex, buttonIndex++, "Dev" )
-		Hud_AddEventHandler( devButton, UIE_CLICK, AdvanceMenuEventHandler( GetMenu( "DevMenu" ) ) )
-	#endif
+	
+	if ( GetConVarBool( "ns_allow_team_change" ) )
```

Comment:
When does this code run? It's on the UI VM so I would assume once, in which case this code won't work as you intend.

Reply:
IIRC Convars replicates from server before map load, though i just found a callback which i could set this up probably.